src/openapi/spec/register-client-schema.ts

```diff
@@ -0,0 +1,39 @@
+import { createSchemaObject, CreateSchemaType } from '../openapi-types';
+
+export const schema = {
+    type: 'object',
+    required: ['appName', 'interval', 'started', 'strategies'],
+    properties: {
+        appName: {
+            type: 'string',
+        },
+        instanceId: {
+            type: 'string',
+        },
+        sdkVersion: {
+            type: 'string',
+        },
+        environment: {
+            type: 'string',
```

Comment:
For descriptions, this property might require some extra attention. Is this the `environment` that the client gets configured via the `environment` option, the environment that its API key applies to, or something else?

Reply:
Only exists for backward compatibility. With newer setups the server will figure out the environment itself from the API key and this one will be discarded. We can and probably should remove it from here and the source (https://github.com/Unleash/unleash/blob/main/src/lib/openapi/spec/client-application-schema.ts)

Reply:
Aye, that's fair. The schema accepts additional properties regardless, so the behavior will stay the same, I suppose.

Reply:
You can also mark the property as deprecated. This will show up in the spec and in the UI, and might be the way to go for now along with a good description of what's up.